runbotics/runbotics-desktop/src/action/microsoft/excel/excel.types.ts

```diff
@@ -161,4 +160,11 @@ export interface WorkbookCell {
     address?: string;
     formula?: string;
     valueType?: string;
-}
\ No newline at end of file
+}
+
+export interface Worksheet {
+    id: string;
+    name: string;
+    position: number;
+    visibility: string;
+}
```

Comment:
do you use it?

Reply:
No, but imo in this case the interface should describe whole object.